### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MishaLehotskyi**
- [DEMO LINK](https://MishaLehotskyi.github.io/layout_stars/)
- [TEST REPORT LINK](https://MishaLehotskyi.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```
Comment (Esceype) [inline src/style.css:13]: What about star in row is this need there ?

**Review by Esceype — CHANGES_REQUESTED** [review]
GJ! Let`s improve your code. Check the following comments bellow

File: src/style.css (hunk 1 above)
Comment (MishaLehotskyi) [reply src/style.css:13]: I don't get what you want me to do(

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,83 @@
...
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+    </div>
+
+    <div class="stars stars--1">
```
Comment (denys-danyliuk) [inline src/index.html:23]:
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>

    <div class="stars stars--1">
```
you should add spaces between multiline sibling blocks

**Review by denys-danyliuk — APPROVED** [review]

**Pull request closed without merging**